client/src/test/java/io/moquette/spi/impl/security/AESTest.java

```diff
@@ -0,0 +1,141 @@
+package io.moquette.spi.impl.security;
+
+import com.diffblue.deeptestutils.mock.DTUMemberMatcher;
+import org.junit.Test;
+import org.junit.Assert;
+import org.junit.runner.RunWith;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+import java.lang.reflect.Method;
+import javax.crypto.Cipher;
+
+import static org.mockito.AdditionalMatchers.or;
+import static org.mockito.Matchers.isA;
+import static org.mockito.Matchers.isNull;
+
+@RunWith(PowerMockRunner.class)
+@PowerMockIgnore({"javax.management.*"})
```

Comment:
Potentially can be more specific here if it's required `@PowerMockIgnore({"javax.management. Cipher"})` but at the same time why would we be ignoring something we are trying to mock? I think this needs further investigation.

Reply:
Mocking has been removed